nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -642,22 +637,32 @@ in
       }
     ];
 
+    systemd.tmpfiles.rules = [
+      "d '${cfg.stateDir}' 0750 ${cfg.user} ${cfg.group} - -"
+      "d '${cfg.stateDir}/logs' 0750 ${cfg.user} ${cfg.group} - -"
```

Comment:
Is this type of quoting supported in tmpfiles rules?

Reply:
Yes, tested in VirtualPC.